student/src/main/java/tds/student/tdslogger/ApplicationShutdownLogger.java

```diff
@@ -0,0 +1,31 @@
+package tds.student.tdslogger;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.ApplicationListener;
+import org.springframework.context.event.ContextClosedEvent;
+import org.springframework.stereotype.Component;
+
+import static tds.student.tdslogger.StudentEventLogger.StudentLogEvent.APP_SHUTDOWN;
+
+@Component
+public class ApplicationShutdownLogger implements ApplicationListener<ContextClosedEvent> {
```

Comment:
I'd be tempted to combine these two classes into a single ApplicationEventLogger